electron_app/src/backup/exporter.js

```diff
@@ -0,0 +1,121 @@
+const path = require('path');
+const {
+  copyDatabase,
+  getFileSizeInBytes,
+  getTempDirectory,
+  removeTempBackupDirectoryRecursive,
+  checkTempBackupDirectory
+} = require('./FileUtils');
+const {
+  zipStreamFile,
+  encryptStreamFile,
+  generateKeyAndIvFromPassword
+} = require('./Compress');
+const { initDatabaseEncrypted, Account } = require('../database/DBEmodel');
+const { exportEncryptDatabaseToFile } = require('./databaseExport');
+const { APP_DOMAIN } = require('../utils/const');
+
+const startTimeout = setTimeout(() => {
+  start();
+}, 5000);
+
+let key;
+let password;
+
+const start = async () => {
+  var args = process.argv.slice(2);
+  const dbPath = args[0];
+  const backupPath = args[1];
+  const recipientId = args[2];
+
+  if (!key) {
+    throw new Error(`Database key was never received`);
```

Comment:
are all the `throw error` are catched and printed using the logger?

Reply:
It is